src/configure/helper/telemetryHelper.ts

```diff
@@ -0,0 +1,66 @@
+import { IActionContext, ITelemetryReporter } from "vscode-azureextensionui";
+
+import { extensionVariables } from "../model/models";
+import { TracePoints } from "../resources/tracePoints";
+
+const uuid = require('uuid/v4');
+
+export class TelemetryHelper {
+    private actionContext: IActionContext;
+    private telemetryReporter: ITelemetryReporter;
+    private journeyId: string;
+    private command: string;
+    constructor(actionContext: IActionContext, command: string) {
+        this.actionContext = actionContext;
+        this.telemetryReporter = extensionVariables.reporter;
+        this.journeyId = uuid();
+        this.command = command;
+        this.setTelemetry(TracePoints.Command, command);
```

Comment:
Shouldn't JourneyId be logged along with command? Otherwise how can I query all JourneyIds for a command?

Reply:
Discussed with Disha and now I understand